securedrop/tests/test_journalist.py

```diff
@@ -1315,10 +1314,7 @@ def tearDown(self):
         utils.env.teardown()
 
     def get_fake_config(self):
-        class Config:
-            def __getattr__(self, name):
-                return getattr(config, name)
-        return Config()
+        return SDConfig()
```

Comment:
thanks to the new **SDConfig** there is no need for **get_fake_config(self)** and all calls to **get_fake_config()** can be replaced with **SDConfig()** instead :+1:

Reply:
I will update the tests in a new PR where will change how we deal with configuration.

Reply:
ok